### PR TITLE
Contact form validation

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project, I did the following :
- When the user wants to submit the form, there is a function to check if the email is in lowercase.
- If the validation is OK, the form is sent.
- If the validation is not OK, an error message is shown to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,4 +25,17 @@ const btn3 = document.getElementById('port');
...
+
+user.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (x.value !== x.value.toLowerCase()) {
+    document.getElementById('error-message').innerText = 'Error, Email must be lowercase. Form is not sent';
+  } else {
+    document.getElementById('error-message').innerText = '';
+    user.submit();
+  }
+});
```
Comment (aamirkhan2478) [inline index.js]: - [ ] Well done :clap:  you did a lovely job :+1: but your code is not followed `DRY` principles kindly use `DRY` principles for more readability and clean code. `DRY` stand for `Don't Repeat Yourself`, a fundamental principle of software development aimed at reducing the repetition of information. check below :point_down:  my suggestions :slightly_smiling_face: 

```suggestion
const form = document.querySelector('#valid');
const emailInput = document.querySelector('#email');
const errorMessage = document.querySelector('#error-message');

function showError(message) {
  errorMessage.innerText = message;
}

form.addEventListener('submit', (event) => {
  event.preventDefault();
  const emailValue = emailInput.value.trim();
  
  if (emailValue !== emailValue.toLowerCase()) {
    showError('Error: Email must be lowercase. Form is not sent.');
  } else {
    showError('');
    form.submit();
  }
});
```

Here's what has been changed:

- Extracted the error message display into a separate function called `showError()` that takes a message parameter.
- Removed the repetition of the `document.querySelector()` calls by assigning them to variables.
- Called the `showError()` function in both branches of the `if` statement, eliminating the duplicated code.
- Used more descriptive variable names to improve readability.
- This makes the code more maintainable and easier to read.

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @tumainimaganiko and @Adidem-dev , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters are passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @tumainimaganiko and @Adidem-dev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (tumainimaganiko) [reply index.js]: I'm happy to understand how the DRY principle works. I'll be sure to keep in mind in future

**Pull request merged**